static/styles/subscriptions.css

```diff
@@ -617,6 +617,10 @@ h4.stream_setting_subsection_title {
     }
 }
 
+#create-stream-simplebar-wrapper {
+    max-height: calc(95vh - 145px);
```

Comment:
This value was found by trial and error. A similar approach was used in other places within this file as well.
(for example: line 224)